inference/core/env.py

```diff
@@ -264,7 +264,7 @@
 
 ONNXRUNTIME_EXECUTION_PROVIDERS = os.getenv(
     "ONNXRUNTIME_EXECUTION_PROVIDERS",
-    "[CUDAExecutionProvider,OpenVINOExecutionProvider,CPUExecutionProvider]",
+    "[CUDAExecutionProvider,OpenVINOExecutionProvider,CoreMLExecutionProvider,CPUExecutionProvider]",
```

Comment:
now everyone not on a mac is gonna get stupid warnings about not having coremlexecution provider :)

Reply:
Yeh I was thinking we should probably update this default to be what `onnxruntime.get_available_providers()` reports but was going to do that in a different PR.